src/components/ArticleList.jsx

```diff
@@ -1,70 +1,107 @@
+        <p className="normalText">Things are happening...</p>
+      </>
+    );
+  }
 
-    if(isError) {
-        console.log(isError)
-        return <>
+  if (isError) {
+    console.log(isError);
+    return (
+      <>
         <img className="errorImage" src=""></img>
-        <p className='normalText'>Status: {isError.status}....Uh Oh</p>
-        </>
-    }
+        <p className="normalText">Status: {isError.status}....Uh Oh</p>
+      </>
+    );
+  }
+  if(viewedArticle){
     return (
-        <section className="ArticleList">
-        <h1>Articles</h1>
-        <nav>
-          <Link className="link" to="/">
-            Home
-          </Link>
-        </nav>
-        <section className='Map'>
+        <section className="JustArticle">
+                      <nav>
+        <Link className="link" to="/">
+          Home
+        </Link>
+      </nav>
+          <h2> {viewedArticle.props.children[1].props.children[1]}</h2>
+          {viewedArticle}
+        </section>
+      );
+  }
+  return (
+    <section className="ArticleList">
+      <h1>Articles</h1>
+      <nav>
+        <Link className="link" to="/">
+          Home
+        </Link>
+      </nav>
+      <section className="Map">
         {allArticles.map((article) => {
-            return(
+          const retArticle = (
+            <section className="SoloArticle" key={article.article_id}>
+              <p>Article id: {article.article_id}</p>
+              <p>Title: {article.title}</p>
+              <p>Topic: {article.topic}</p>
+              <p>Author: {article.author}</p>
+              <p>Created_at: {article.created_at}</p>
+              <p>Votes: {article.votes}</p>
+              <p>Comment count: {article.comment_count}</p>
+              <img src={article.article_img_url} alt="image" />
+            </section>
+          );
+          return (
             <section className="Article" key={article.article_id}>
-                <p>Article id: {article.article_id}</p>
-                <p>Title: {article.title}</p>
-                <p>Topic: {article.topic}</p>
-                <p>Author: {article.author}</p>
-                <p>Created_at: {article.created_at}</p>
-                <p>Votes: {article.votes}</p>
-                <p>Comment count: {article.comment_count}</p>
-                <img src={article.article_img_url} alt='image'/>
-        </section>
-            )
+              <p>Article id: {article.article_id}</p>
+              <p>Title: {article.title}</p>
+              <p>Topic: {article.topic}</p>
+              <p>Author: {article.author}</p>
+              <p>Created_at: {article.created_at}</p>
+              <p>Votes: {article.votes}</p>
+              <p>Comment count: {article.comment_count}</p>
+              <img src={article.article_img_url} alt="image" />
+              <button
+                onClick={() => onClick(retArticle)}
+              >
+                View Article
+              </button>
```

Comment:
I like the idea of this, but this is not the way we should be displaying the single article. This should be done by using `<Link...>` tag which should navigates to another page which displays only the single article that was selected.